### DISCUSSION TITLE
What is the recommended storage for persistence in React Native?

### DISCUSSION THREAD

**Discussion by KrisLau** [pmndrs/zustand#1533] [General] [answered]
Would you recommend using `localStorage` (not sure if this is available in React Native/mobile apps), `AsyncStorage` or something like this: https://nozbe.github.io/WatermelonDB/Advanced/LocalStorage.html?

Also, best practice wise if I have a lot stores to initialize should I initialize them all in the App.js?

**Comment by dai-shi** [comment, marked as answer]
I think many people use one of AsyncStorage libs: https://reactnative.dev/docs/asyncstorage

Someone says they use this one: https://github.com/mrousavy/react-native-mmkv

Hope it helps.
